src/Inciter/Performer.h

```diff
@@ -89,6 +89,28 @@ class Performer : public CBase_Performer {
 
     //! Advance equations to next stage in multi-stage time stepping
     void advance( uint8_t stage, tk::real dt, uint64_t it, tk::real t );
+  
+    //! Create PUP routine to prepare Performer chare for object migration
+		void pup(PUP::er &p) {
+      //! Call PUP routine for superclass
+      CBase_Performer::pup(p);
```

Comment:
With the latest Charm++ version, I don't believe this is required anymore. But need to test it somehow.